Extend 2Dim embedding bag benchmarking to include 3Dim benchmarks

EdwaRen opened the pull request:
Summary: Add support for benchmarking of 8 bit quantizations of N-D batched embeddings. Currently only works for 3Dim embeddings and still requires thought on ramping up from 3Dim to NDim.

Test Plan: ```buck run //caffe2/benchmarks/operator_benchmark/pt:qembedding_pack_test```

Differential Revision: D30770085

jingsh reviewed (commented):
can you fix the failed lint check?

let's go with what we discussed on the diff, do two diffs, first add tests, and second to clean up

EdwaRen commented:
> can you fix the failed lint check?
> 
> let's go with what we discussed on the diff, do two diffs, first add tests, and second to clean up

Sounds good to me!

EdwaRen committed:
Extend 2Dim embedding bag benchmarking to include 3Dim benchmarks (#64647)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/64647

Add support for benchmarking of 8 bit quantizations of N-D batched embeddings. Currently only works for 3Dim embeddings and still requires thought on ramping up from 3Dim to NDim.

Test Plan: ```buck run //caffe2/benchmarks/operator_benchmark/pt:qembedding_pack_test```

Differential Revision: D30770085

fbshipit-source-id: e46bf09057347dd9d391d2d4f7468b7a0c46aabc
